cabal.project

```diff
@@ -46,3 +46,11 @@ write-ghc-environment-files: always
 -- IMPORTANT
 -- Do NOT add more source-repository-package stanzas here unless they are strictly
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
+
+source-repository-package
```

Comment:
I think that's not needed anymore?

Reply:
The change on the tag came after the tag of 9.0.0.0, so we will need a new api release for this. Without the SRP it fails to build. I'll remove SRP and merge after we have a new api.  

![Screenshot_20240704_084257](https://github.com/IntersectMBO/cardano-cli/assets/64222734/04cc4c86-13c4-4146-802d-0a0f49fb0c4c)